### PR TITLE
Whatsnew

### PR DESCRIPTION
**Pull request by jburel**
Replaces https://github.com/openmicroscopy/ome-documentation/pull/1756 
The original PR was blocking https://github.com/openmicroscopy/ome-documentation/pull/1761 to go in

cc @hflynn 

----

See https://trello.com/c/UloMh1fl/108-document-new-features-in-whats-new-pages-and-on-website

This adds content for Users and Sysadmins and also highlights the CLI docs from the top index page and the User index page ( @rleigh-codelibre pointed this issue out - the change to the 'About the OMERO Platform' heading dated from all the user docs being moved to help but now we have CLI docs here that are basically hidden unless you read everything really thoroughly!)

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-docs#844](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/844/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/844/consoleText) for more details.
Possible conflicts:
  - PR #1760 jburel 'http to https'
    - omero/users/whatsnew.rst

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-docs#845](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/845/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/845/consoleText) for more details.
Possible conflicts:
  - PR #1760 jburel 'http to https'
    - omero/users/whatsnew.rst

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-docs#846](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/846/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/846/consoleText) for more details.
Possible conflicts:
  - PR #1760 jburel 'http to https'
    - omero/users/whatsnew.rst

**Comment by joshmoore** [comment]
@jburel : are the TODOs from the original PR taken care of?

**Comment by jburel** [comment]
@joshmoore no that's what I mentioned this morning
What's new for dev needs to be done, but it can be done in another PR
@will-moore was going to check the first one
The second one can be handled in another PR

**Comment by will-moore** [comment]
Opened PR: https://github.com/openmicroscopy/ome-documentation/pull/1767

**Comment by jburel** [comment]
Second TODOs: https://github.com/openmicroscopy/ome-documentation/pull/1769

**Comment by pwalczysko** [comment]
I thnk I am missing mention of 2 features here, the "Create P/D/S for others" in web and "Chown all belonging to user" see my comment on https://github.com/openmicroscopy/ome-documentation/pull/1756#issuecomment-330802041

**Comment by pwalczysko** [comment]
Builds and reads well. 

Other than https://github.com/openmicroscopy/ome-documentation/pull/1765#issuecomment-330802296 (2 missing features) all fine here. I think that the two features should really be mentioned, as I cannot find them in the version histories either.

**Comment by jburel** [comment]
@pwalczysko: I think the 2 features mentioned in https://github.com/openmicroscopy/ome-documentation/pull/1765#issuecomment-330802296
are covered with the general text.
```
This is a way to
  cater for the needs for some trusted users to act on behalf of all the OMERO
  users in a facility, with access to all the groups and data in OMERO.
```
I don't think we need to be a specific as create P/D by light admin

**Comment by jburel** [comment]
There is a version history PR currently opened https://github.com/openmicroscopy/openmicroscopy/pull/5473
 ``Chown all`` is missing from the history. I will add it

**Comment by will-moore** [comment]
Previously, even a Full Admin could not create P/D/S for another user (except by also doing a Chgrp!), so I think this is worth mentioning.

**Comment by pwalczysko** [comment]
@will-moore  is right here - the feature is about creating something in Webclient for others (completely new in 5.4.0, for light admins and sysadmins), this always was a highly demanded feature by the users

**Comment by jburel** [comment]
@pwalczysko @will-moore better?

**Comment by pwalczysko** [comment]
@jburel : Yes, this looks fine to me, thank you.

**Comment by will-moore** [comment]
Just thought - for Sysadmins we can add https://github.com/openmicroscopy/openmicroscopy/pull/5463 and maybe bug-fix https://github.com/openmicroscopy/openmicroscopy/pull/5485 
and for users, bug-fix https://github.com/openmicroscopy/openmicroscopy/pull/5441

File: omero/users/whatsnew.rst [hunk 1]
```diff
@@ -1,35 +1,23 @@
...
+  without having to be granted full administrator privileges. This is a way to
+  cater for the needs for some trusted users to act on behalf of all the OMERO
+  users in a facility, with access to all the groups and data in OMERO. Full administrators and administrators with restricted privileges can now create Project/Dataset/Screen on behalf of other users in OMERO.web or transfer all the data of a given user using the :program:`chown` command.
 
-See the :help:`User help website <>` for information on how to incorporate
-these new features into your current workflows. There is also a
-`short demo movie <https://downloads.openmicroscopy.org/movies/omero-5-3/mov/OMERO.web-5.3.0.mov>`_ for the new features of OMERO.web.
+- Improvements to OMERO.web including the display of plates and wells, the
+  synchronization of controls enabling copy/paste of rendering settings, the display of 32-bit and float images, the layout of the left-hand panel and the
+  ability to export images in plate as OME-TIFF.
```
Comment (will-moore) [inline omero/users/whatsnew.rst]: I don't remember what the 'synchronization of controls' was. The left-hand panel looks almost identical (DOM & css cleanup). The fix for float images is only noticeable for images with small range.

I would say something like: "Other improvements to OMERO.web include the display of fields within Wells, handling of float images with small dynamic range and the ability to export images in plate as OME-TIFF as well as a number of minor bug fixes." ?

**Comment by jburel** [comment]
@will-moore adjusted as suggested

**Comment by will-moore** [comment]
Looks good, thanks

**Pull request merged**